SchedulePlatform.Api/Models/MenuPatchModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using SchedulePlatform.Models.Enums;
+
+namespace SchedulePlatform.Api.Models
+{
+	public class MenuPatchModel
+	{
+        public WeekDays? Day { get; set; }
```

Comment:
Please check the indentation of the code everywhere. I won't add any more comment on this subject as I've added already too many. 😄